admin-tools/lib/src/main/scala/com/gu/mediaservice/indexing/package.scala

```diff
@@ -12,7 +12,9 @@ package object indexing {
     val Finished = ProduceProgress("finished", 3)
     val Reset = ProduceProgress("reset because of failure", 0)
     val KnownError = ProduceProgress("blacklisted because of known failure", 4)
-
+    val Locating = ProduceProgress("looking for image",5)
+    val Found = ProduceProgress("image found", 6)
+    val Inconsistent = ProduceProgress("image not found", 7)
```

Comment:
```suggestion
    val notFound = ProduceProgress("image not found", 7)
```

Reply:
https://github.com/guardian/grid/pull/2846/files/1c5f2ca407dd44fe2e8b6a4b13ce050eb012392d#diff-09da8df871628152c10a83b90ed3e022R11

Reply:
Following the conversation in an above thread, this might be more explicit. However, just noticed that the message here is unique so this is just a minor comment.

```suggestion
    val Inconsistent = ProduceProgress("re-ingested image not found in media-api", 7)
```